ui/hooks/multi-srp/useHdKeyringsWithSnapAccounts.test.ts

```diff
@@ -67,6 +77,9 @@ describe('useHdKeyringsWithSnapAccounts', () => {
       mockFirstPartySnapAccount,
       mockSnapAccount2,
     ]);
+    mockIsSnapPreinstalled.mockImplementation(
+      (prop: string) => Boolean(prop) !== undefined,
```

Comment:
IMO, we could just go with `true` for now, we don't need the `!== undefined` anymore if we go with my other suggestion.

```suggestion
      () => true,
```